recipes/pysolr/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pysolr" %}
+{% set version = "3.6.0" %}
```

Comment:
set `setup.py` has the correct version "3.6.0". The `__version__` was not adapted ... but i have looked in the github code and it was fixed after the 3.6.0 release:

https://github.com/django-haystack/pysolr/blob/master/pysolr.py#L61